_posts/events/2019-09-27-mumttt.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+categories: [event]
+event_name: "The 4th Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2019)"
+event_url: "http://www.lexytrad.es/europhras2019/mumttt-2019-2/"
+event_location: "Malaga, Spain"
+event_submission_date: "2019-06-10"
+host_name: "europhras 2019"
+host_url: "http://www.lexytrad.es/europhras2019/"
+siglex_endorsed: yes
+---
+The 4th Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2019)
+
+Malaga, 27 September 2019
```

Comment:
You probably want two spaces at the end of this line so that the URL shows up on its own line.

Reply:
Done